llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -918,6 +918,16 @@ static void simplifyRecipe(VPRecipeBase &R, VPTypeAnalysis &TypeInfo) {
     // value with the others blended into it.
 
     unsigned StartIndex = 0;
+    for (unsigned I = 0; I != Blend->getNumIncomingValues(); ++I) {
+      // If a value's mask is only used by the blend then is can be deadcoded.
```

Comment:
```suggestion
      // If a value's mask is used only by the blend then is can be deadcoded.
```

Reply:
Done, but this feels artificial given both are legitimate english.